client/src/sessions/mod.rs

```diff
@@ -1,48 +1,31 @@
-use arrayvec::ArrayVec;
-use tpm2_rs_base::errors::{TssResult, TssTcsError};
-use tpm2_rs_base::{
-    Tpm2bAuth, Tpm2bNonce, Tpm2bSimple, TpmaSession, TpmiShAuthSession, TpmsAuthCommand,
-    TpmsAuthResponse,
-};
+//! This module contains collection of traits setting lower bounds
+//! on how many sessions are allowed in an authorization area.
+//! In particular the this module defines the following traits
+//! - [`AuthorizationArea`]: A trait for authorization area with
+//!   (possibly zero) unkown number of sessions up to three.
+//! - [`AuthorizationArea1Plus`]: A trait for authorization area
+//!   with at least one session and at most three sessions
+//! - [`AuthorizationArea2Plus`]: A trait for authorization area
+//!   with either two or three sessions.
+//!
+//! The concept of a session is defined by [`Session`].  
+// TODO we may need more than that & This is intentionally left out of cargo docs
```

Comment:
I don't know what this line is trying to say, did you intend to leave this comment?